README.md

```diff
@@ -1,50 +1,37 @@
-DNS query functions
-===================
+## Table of Contents
 
-This module contains query functions for DNS for use from Puppet.
+1. [Overview](#overview)
+1. [Setup - The basics of getting started with dnsquery](#setup)
+1. [Usage - Configuration options and additional functionality](#usage)
+1. [Development - Guide for contributing to the module](#development)
 
-Usage
------
+## Overview
 
-All functions can take a optional second lambda argument that is called if the lookup returned no values. Then the result of the lambda will be used instead.
+This module provides functions and facts for querying dns records
 
-### dns_lookup
+## Setup
 
-Does a DNS lookup and returns an array of addresses.
+```console
+ % puppet module install puppet-dnsquery
```

Comment:
Commonly `$` is used for non-root and `#` for root. I don't see `%` that often for consoles.

Reply:
indeed not many people using the c shell theses days :), updated